api/interfaces/interfaces.go

```diff
@@ -61,22 +61,59 @@ type ZoneWithPopulatedLoomies struct {
 	PopulatedLoomies []WildLoomie         `json:"populated_loomies" bson:"populated_loomies"`
 }
 
+type GymRewardItem struct {
+	RewardCollection string             `json:"reward_collection" bson:"reward_collection"`
+	RewardId         primitive.ObjectID `json:"reward_id" bson:"reward_id"`
+	RewardQuantity   int                `json:"reward_quantity" bson:"reward_quantity"`
+}
+
 type Gym struct {
-	Id        primitive.ObjectID `json:"_id" bson:"_id"`
-	Latitude  float64            `json:"latitude"      bson:"latitude"`
-	Longitude float64            `json:"longitude"      bson:"longitude"`
-	Name      string             `json:"name"      bson:"name"`
+	Id                    primitive.ObjectID   `json:"_id" bson:"_id"`
+	Latitude              float64              `json:"latitude"      bson:"latitude"`
+	Longitude             float64              `json:"longitude"      bson:"longitude"`
+	Name                  string               `json:"name"      bson:"name"`
+	Owner                 primitive.ObjectID   `json:"owner,omitempty"      bson:"owner,omitempty"`
+	CurrentPlayersRewards []GymRewardItem      `json:"current_players_rewards"      bson:"current_players_rewards"`
+	CurrentOwnerRewards   []GymRewardItem      `json:"current_owner_rewards"      bson:"current_owner_rewards"`
```

Comment:
The owner cannot get his rewards, the returning array is null.
![image](https://user-images.githubusercontent.com/61242172/224494907-8b5952d2-a3e3-40c5-bbf4-675f2a6aabbe.png)

To fix it, the property `current_owner_rewards` needs to be renamed to `current_owners_rewards`.

Reply:
Done